src/MLJInterface.jl

```diff
@@ -157,7 +157,7 @@ function mlj_to_kwargs(model::LGBMClassifier, classes)
 end
 
 # X and y and w must be untyped per MLJ docs
-function fit(mlj_model::MODELS, verbosity::Int, X, y, w=Vector{AbstractFloat}())
+function fit(mlj_model::MODELS, verbosity::Int, X, y, w=AbstractFloat[])
```

Comment:
Just a style change? If so, you have others.

Reply:
`fit.jl` 30-31

Reply:
Thanks -- something about `ArrayType{Type}(initialise)` grinds my gears and i find `TypeToArray[]` much cleaner

Reply:
25ee0d4